support-frontend/assets/helpers/campaigns.jsx

```diff
@@ -18,46 +22,46 @@ export type CampaignSettings = {
   goalReachedCopy?: React$Element<string>, // If set, the form will be replaced with this if goal reached
 };
 
-export type Campaigns = {
-  [string]: CampaignSettings,
-};
-
-const currentCampaignName = ausMomentEnabled('AU') ? 'au/contribute' : null;
+const currentCampaignPath: string | null = ausMomentEnabled('AU') ? 'au/contribute' : null;
 
-export const campaigns: Campaigns = currentCampaignName ? {
```

Comment:
The `campaigns` object was creating unnecessary compexity. I've replaced it with a single `campaign` const, which is returned by `getCampaignSettings()` if the user is on the right url and the campaign is switched on